base_classes/NXunit_cell.nxdl.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+#
+# Copyright (C) 2023-2025 NeXus International Advisory Committee (NIAC)
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXunit_cell" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <symbols>
+        <symbol name="d">
+            <doc>
+                Dimensionality of the lattice.
+            </doc>
+        </symbol>
+    </symbols>
+    <doc>
+        Base class to describe structural aspects of an arrangement of
+        atoms or ions including a crystallographic unit cell.
+        
+        Following recommendations of `CIF &lt;https://www.iucr.org/resources/cif/spec/version1.1&gt;`_ and `International Tables of Crystallography &lt;https://it.iucr.org/&gt;`_.
+    </doc>
```

Comment:
This base class should be reviewed by a crystallography person like @phyy-nx and @yayahjb